### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by jmikell821**
Resolves first attempt of #4229. 

Preview: https://github.com/elastic/security-docs/blob/doc-template-new/.github/ISSUE_TEMPLATE/internal-documentation-request--elastic-employees-.yaml
(It's not a perfect representation but it gives an idea of what the form will actually look like.)

### CODE REVIEW

**Comment by joepeeples** [comment]
Empty placeholder elements were causing validation errors, so I removed them for now. We can add them back once we have actual placeholder text; this was just to get the form to build correctly.

**Comment by joepeeples** [comment]
Now that we can [preview the form](https://github.com/elastic/security-docs/blob/doc-template-new/.github/ISSUE_TEMPLATE/internal-documentation-request--elastic-employees-.yaml), overall it's pretty long, and could feel overwhelming or even discourage folks from creating issues. I think we could streamline it by combining a few fields, but it's hard to convey large-scale suggestions with GitHub's line-edit review UI. So I created an alternate PR (https://github.com/elastic/security-docs/pull/4308) to show one way we could trim down without losing content.

Maybe we could discuss in our next team sync?

**Review by benironside — APPROVED** [review]
One minor edit, otherwise LGTM

File: .github/ISSUE_TEMPLATE/internal-documentation-request--elastic-employees-.yaml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        What: We're introducing new feature A.
+        When: This feature will launch at the completion of project B.
+        Why: This feature will make X, Y, and Z easier for the user.
+    validations:
+      required: true
+  - type: textarea
+    id: related
+    attributes:
+      label: Resources
+      description: Please include the relative pull request(s) or issue(s).
```
[the remark this answers is not in the document]
Comment (joepeeples) [reply .github/ISSUE_TEMPLATE/internal-documentation-request--elastic-employees-.yaml:24]: @benironside This was fixed in the [streamlined version](https://github.com/elastic/security-docs/pull/4308) of the template form.

**Comment by joepeeples** [comment]
We updated the template form when we merged https://github.com/elastic/security-docs/pull/4308, so we can close this PR.

**Pull request closed without merging**